src/components/Footer/Footer.tsx

```diff
@@ -81,6 +81,14 @@ const DetailsStyles = styled.details`
       transform: rotate(180deg);
     }
   }
+
+  @media (max-width: 900px) {
```

Comment:
Please use/add helper
https://onja20.slack.com/archives/C01BXQRM6LV/p1627044969006300
https://codesandbox.io/s/media-queries-in-styled-components-typescript-forked-o4tt0